spec/models/dynamic_field_group_spec.rb

```diff
@@ -317,4 +317,36 @@
       expect(dynamic_field_group.siblings).to match_array []
     end
   end
+
+  describe '#path' do
+    it 'is derived from ancestor path' do
+      expect(dynamic_field_group.path).to eql("#{dynamic_field_group.parent&.path}/#{dynamic_field_group.string_key}")
+    end
+   it 'is unique' do
+      expect {
+        FactoryBot.create(:dynamic_field_group, string_key: dynamic_field_group.string_key, parent: dynamic_field_group.parent)
+      }.to raise_error
```

Comment:
When possible, I like to do more specific error class matching.  Rubocop may also complain if you don't.

Reply:
I'll make it more specific as soon as a I know what the error will be 😂

Reply:
Sounds good, haha.

Reply:
I think this is addressed in the PR now.